### DISCUSSION TITLE
[FYI] KDTree blocker from dblink

### DISCUSSION THREAD

**Discussion by NickCrews** [moj-analytical-services/splink#2297] [General]
I just wanted to point out the blocking algorithm that I ran into that `dblink` uses. It sounds sorta like the auto blocking rules that you implemented in a previous PR here @RobinL ? Perhaps of interest, or you could reach out and see what lessons they learned from doing that auto blocking.

https://github.com/cleanzr/dblink/blob/dc3dd0daf55f8a303863423817a0f0042b3c275a/docs/configuration.md#partitioner

**Comment by RobinL** [comment]
Thank you! will take a look.

Yeah - interesting to see it appears to be quite a similar strategy to what I came to (if i understand it correctly!)
